README.md

```diff
@@ -1,3 +1,6 @@
+## API documentation (Swagger)
+Open [http://localhost:3000/api/docs](http://localhost:3000/api/docs) to view it in the browser.
```

Comment:
Link wrong

Reply:
Wrong if you use our nginx proxy, but correct if not